docs/tutorial/pre-requisites-of-running-on-chain-spam-protected-chat2.md

```diff
@@ -16,96 +19,89 @@ If you already have an Ethereum account with sufficient ethers on the Sepolia te
     
     ![](https://i.imgur.com/zFduIV8.jpg)
 
-4. You should already see an account created. As you can see on top right, it should be pointing to Ethereum mainnet.
+4 The automatically created account points to the Ethereum Mainnet by default (dropdown menu in the top right corner).
```

Comment:
I'm not sure an account can point to a network. Maybe something like "by default, Metamask connects to the Ethereum Mainnet"?

Reply:
Indeed. Fixed in https://github.com/waku-org/nwaku/pull/2206/commits/4d921d9426bd22664ee3cf19ea355fb1219fe890